lib/node.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  idb: function () {
```

Comment:
`idb` is a function here, but in the browser version it's an object with two methods.